content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -219,12 +220,29 @@ The easiest way to use the Kubernetes API from a Pod is to use
 one of the official [client libraries](/docs/reference/using-api/client-libraries/). These
 libraries can automatically discover the API server and authenticate.
 
+#### Using Official Client Libraries
+
+From within a Pod, the recommended ways to connect to the Kubernetes API are:
+
+  - For a Go client, use the official [Go client library](https://github.com/kubernetes/client-go/).
```

Comment:
Despite what this page says, there are actually 6 [official client libraries](https://kubernetes.io/docs/reference/using-api/client-libraries/#officially-supported-kubernetes-client-libraries); in the future, there may be more. I wouldn't like to only document this for a subset of those 6, nor does it feel like a good idea to add instructions _here_ for all 6.

Maybe there is a better place to explain how to do in-cluster configuration for each official client library? A different task page perhaps?

Reply:
The section about connecting to the API has always only mentioned 2 official client libraries.  I know there's another page that lists dozens of other libs, but I'm not in that business right now.  Today I needed to obtain Openshift Routes info, but since this is not part of the Python k8s API I was forced to connect to the REST interface directly;  I had to put in a bit of effort to figure that out and this is the only place where it's (briefly) documented.  Once I'd done that, I took one of the client "external API connection" examples from further up the page, modified it for the "in-pod" use-case and added it to this bottom section.  The existing paragraphs are a bit repetetive/poorly written and I moved them around slightly to improve the structure, with the new example at the end.